examples/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[workspace]
```

Comment:
Also we'll have to update the `source` in each example's `spin.toml` to point at `target/` in this workspace.

Reply:
Good catch!